mava/wrappers/jaxmarl.py

```diff
@@ -168,7 +168,7 @@ def __init__(
         self,
         env: MultiAgentEnv,
         has_global_state: bool,
-        timelimit: int,
+        timelimit: int = 1000,
```

Comment:
This won't be unified for Jaxmarl wrapper? (time_limit instead)

Reply:
Had to do this because all children of MultiAgentEnv don't require timelimit to be passed (it's optional), so I just supplied a default. I can make it `-1` and then add an error if it's `< 0`